azurelinuxagent/common/cgroupapi.py

```diff
@@ -209,15 +209,21 @@ def create_extension_cgroups_root(self):
         """
         Creates the directory within the cgroups file system that will contain the cgroups for the extensions.
         """
+        cgroups = []
```

Comment:
Do we ever need these cgroups? (for the root directory of all extensions?)

Reply:
No. Removed this part. I needed it for some testing.